src/DonationForms/V2/resources/components/DonationFormsListTable.tsx

```diff
@@ -63,6 +63,13 @@ const donationStatus = [
     },
 ];
 
+const urlParams = new URLSearchParams(window.location.search);
+urlParams.get('id');
```

Comment:
I guess this is a leftover?

Reply:
Yep! Resolve in 93621fbedf84ce0f43ba9280373e0ac12c0f9881